Userland/Applications/Piano/RollWidget.cpp

```diff
@@ -187,7 +187,7 @@ void RollWidget::mousedown_event(GUI::MouseEvent& event)
         return;
 
     if (event.button() == GUI::MouseButton::Secondary) {
-        auto const time = roll_length * (static_cast<double>(get_note_for_x(event.x())) / m_num_notes);
```

Comment:
Shouldn't `get_note_for_x` be fixed instead?

Reply:
Maybe, but this won't fix the case when a RollNote is not the full width of Piano note, like here:

![image](https://user-images.githubusercontent.com/42967349/236871365-5b031b74-6094-411a-b2bb-f1136ff962dc.png)

The only other place where `get_note_for_x` is used is for placing notes, where it works well (????).

Reply:
Regardless, either we fix `get_note_for_x` since that function sounds suspiciously close to what you're trying to achieve, or we drop it altogether. Leaving it in the way it is now is just asking for future problems :^)

Reply:
Now I replaced get_note_for_x() with get_sample_for_x() which is just rounded to note boundary when placing, while deletion uses samples.